spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -290,15 +290,15 @@
       let(:phone_confirmed) { false }
 
       it 'prompts to finish verifying profile, then redirects to SP' do
-        allow(FeatureManagement).to receive(:reveal_usps_code?).and_return(true)
-
+        allow(FeatureManagement).to receive(:reveal_usps_code?).and_return true
```

Comment:
What do you think about removing these changes since they are cosmetic, to keep this PR focused on the addition of the feature flag?